CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -417,7 +417,8 @@ public function import($values): void {
         $error = $this->checkContactDuplicate($paramValues);
 
         if (CRM_Core_Error::isAPIError($error, CRM_Core_ERROR::DUPLICATE_CONTACT)) {
-          $matchedIDs = explode(',', $error['error_message']['params'][0]);
+          $matchedIDs = $error['error_message']['params'];
+          $matchedIDs = is_array($matchedIDs) ? $matchedIDs : [$matchedIDs];
```

Comment:
Nothing wrong with this, but could also be expressed more succinctly as:

```
$matchedIDs = (array) $error['error_message']['params'];
```

Reply:
@MegaphoneJon if you use the suggestion button then your suggestion can be committed (but you have to select both lines before you make your comment).

Reply:
Thanks @MegaphoneJon. I incorporated this suggestion.